data/1.16.220/protocol.json

```diff
@@ -422,7 +422,7 @@
                       {
                         "compareTo": "network_id",
                         "fields": {
-                          "355": [
+                          "\"||global.ShieldItemID||355||\"": [
```

Comment:
This is a bad hack that exploits how the compiler works,
```js
              case ""||global.ShieldItemID||355||"": return ((buffer, offset) => {
```

so we can get a JS variable inside. Could also make some changes to protodef, although I'm not sure how that would work.